src/main/pg/displayport_profiles.h

```diff
@@ -30,6 +30,9 @@ typedef struct displayPortProfile_s {
     uint8_t whiteBrightness;
     int8_t displayPortSerial;  // serialPortIdentifier_e
 
+    // For attribute-rich OSDs
+
+    uint8_t attrValues[4];     // NORMAL, INFORMATIONAL, WARNING, CRITICAL
 #ifdef USE_DISPLAYPORT_MSP_VENDOR_SPECIFIC
```

Comment:
Just noticed this, and not relevant for this pull request, but important for future pull requests: Structs used in parameter groups can not have conditional components, otherwise the invariant that the structure of a parameter group is fully defined for any given name / id and version is violated.